src/patterns/OCCollaborationStatus.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div class="oc-collaboration-status">
```

Comment:
The component itself is find - but to me this is something to be implemented in phoenix.